keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -243,49 +231,46 @@ A `Bucket` is not deleted if it is bound to a `BucketRequest`.
 
 #### BucketClass
 
-An immutable, cluster-scoped, custom resource to provide admins control over the handling of bucket provisioning.  The `BucketClass` (BC) defines a retention policy, driver specific parameters, and the provisioner name. A list of allowed namespaces can be specified to restrict new bucket creation and access to existing buckets. A default bucket class can be defined for each supported protocol. This allows the bucket class to be omitted from a `BucketRequest`. Relevant `BucketClass` fields are copied to the `Bucket` instance to handle the case of the BC being deleted or re-created.  If an object store supports more than one protocol then the admin should create a `BucketClass` per protocol.
+A cluster-scoped, custom resource to provide admins control over the handling of bucket provisioning.  The `BucketClass` (BC) defines a retention policy, driver specific parameters, and the provisioner name. A list of allowed namespaces can be specified to restrict new bucket creation and access to existing buckets. A default bucket class can be defined for each supported protocol. This allows the bucket class to be omitted from a `BucketRequest`. Relevant `BucketClass` fields are copied to the `Bucket` instance to handle the case of the BC being deleted or re-created.  If an object store supports more than one protocol then the admin should create a `BucketClass` per protocol.
+
+A `BucketClass` is necessary for greenfield and "static" (driverless) brownfield use cases. It is not needed for dynamic brownfield since all of the information in the BC already exists in the `Bucket` instance.
 
-NOTE: the `BucketClass` object is immutable except for the field `isDefaultBucketClass`
+> Note: the `BucketClass` object is immutable except for the `isDefaultBucketClass` annotation.
 
 ```yaml
 apiVersion: cosi.io/v1alpha1
 kind: BucketClass
 metadata:
   name: 
-provisioner: [1]
-isDefaultBucketClass: [2]
+  annotation:
+    isDefaultBucketClass: [1]
+provisioner: [2]
 protocol:
   name: [3]
   version: [4]
```

Comment:
Version field here has no value and should be removed.
1. There is no way for COSI drivers/plugins to generate this field
2. Clients would not consume this field

Reply:
I have already removed _AnonymousAccessMode_ from the `BucletClass` and the `Bucket` for the same reasons, so this seems like a reasonable and consistent request. Any other opinions?